### ISSUE TITLE
it seems  hard to reproduce your results in paper.

### ISSUE THREAD

**Issue by patrick-tssn** [ictnlp/DSTC8-AVSD#4]
I tried to grid search the params in generate program, but the results can't achieve which in your paper. May I ask if it possible to share your SOTA CKPTS ?

**Comment by patrick-tssn** [comment]
The best score on DSTC8-AVSD so far (LM + VASM)
CIDEr: 1.220
Bleu_4: 0.434
Bleu_3: 0.517
Bleu_2: 0.617
Bleu_1: 0.738
ROUGE_L: 0.589
METEOR: 0.284

**Comment by lizekang** [comment]
Hello, It's hard to find the checkpoint. I will help you to reproduce it. Which checkpoint do you select in your experiments?

**Comment by patrick-tssn** [comment]
thanks for your reply, the parameters of generate model are: ckpt:114888 beam_size: 4 min_len: 1 penalty: 0.3. I test it on dstc8avsd test set. training procedure is same with your released code.

**Comment by lizekang** [comment]
Which epoch checkpoint did you select? You should select the checkpoint at epoch 3 or 4.  For the other hyper-parameters, you can keep same with the code.

**Comment by patrick-tssn** [comment]
I can obtain comparable score now, thank you for your reply.
